cpp/modmesh/toggle/RadixTree.cpp

```diff
@@ -0,0 +1,132 @@
+#include <iostream>
+#include <memory>
+#include <list>
+#include <algorithm>
+
+class TimedEntry {
+public:
+    size_t count() const { return m_count; }
+    double time() const { return m_time; }
+
+    TimedEntry& add_time(double time) {
+        ++m_count;
+        m_time += time;
+        return *this;
+    }
+
+private:
+    size_t m_count = 0;
+    double m_time = 0.0;
+};
```

Comment:
Always add `/* end class TimedEntry */`.  It's important when we have to hack without an IDE.  It is often that we cannot use an IDE.

Reply:
Done.